### PR TITLE
Phase1/feature/tb1 003 implement add operation 1

### PR DESCRIPTION
**Pull request by romofthez**

### CODE REVIEW

File: CloudCulator/Cloudculator.Core.UnitTests/CalculatorEngineUnitTests.cs [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+namespace Cloudculator.Core.UnitTests
+{
+    public class CalculatorEngineUnitTests
+    {
+        private CalculatorEngine service;
+
+        public CalculatorEngineUnitTests()
+        {
+            service = new CalculatorEngine();
```
Comment (Duch003) [inline CloudCulator/Cloudculator.Core.UnitTests/CalculatorEngineUnitTests.cs]: Since it is private field, please use underscore notation: _service

File: CloudCulator/Core/Class1.cs [hunk 2]
```diff
@@ -1,7 +0,0 @@
-﻿namespace Core
-{
-    public class Class1
-    {
```
Comment (Duch003) [inline CloudCulator/Core/Class1.cs:4]: Is this file needed?

**Review by Duch003 — CHANGES_REQUESTED** [review]

File: CloudCulator/Core/Class1.cs (hunk 2 above)
Comment (romofthez) [reply CloudCulator/Core/Class1.cs:4]: This file no longer exists

File: CloudCulator/Cloudculator.Core.UnitTests/CalculatorEngineUnitTests.cs (hunk 1 above)
Comment (romofthez) [reply CloudCulator/Cloudculator.Core.UnitTests/CalculatorEngineUnitTests.cs]: Done
